modules/transports/core/nhttp/src/test/resources/org/apache/synapse/transport/certificatevalidation/certificates/github/DigiCertHighAssuranceEVCA-1

```diff
@@ -1,28 +1,28 @@
 -----BEGIN CERTIFICATE-----
-MIIEtjCCA56gAwIBAgIQDHmpRLCMEZUgkmFf4msdgzANBgkqhkiG9w0BAQsFADBs

```

Comment:
Do we need to update the existing certificates in the master branch?

Reply:
else test will fail